src/sidebar/components/MarkdownEditor.tsx

```diff
@@ -280,6 +282,27 @@ function TextArea({
     [onEditText, textareaRef],
   );
 
+  const usersListboxId = useId();
+  const accessibilityAttributes = useMemo((): TextareaHTMLAttributes => {
+    if (!popoverOpen) {
+      return {};
+    }
+
+    const selectedSuggestion = userSuggestions[highlightedSuggestion];
+    const activeDescendant = selectedSuggestion
+      ? `${usersListboxId}-${selectedSuggestion.username}`
+      : undefined;
+
+    return {
+      role: 'combobox',
+      'aria-controls': usersListboxId,
+      'aria-expanded': true,
+      'aria-autocomplete': 'list',
+      'aria-haspopup': 'listbox',
+      'aria-activedescendant': activeDescendant,
```

Comment:
These attributes follow the instructions in the [`aria-autocomplete`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-autocomplete#description) MDN docs. Might be worth adding a link to those.